Documentation/Signoffs/SteeringSystem.md

```diff
@@ -40,11 +42,15 @@ _Rotary Encoder Pinout_
 
 _Arduino UNO Pinout_
 
+![image](https://user-images.githubusercontent.com/100802413/202334812-ddca2adf-fbd7-4da2-900d-f3958cb5c812.png)
```

Comment:
This is not third party buildable. The signals being connected need to be enumerated and every connection shown. Also, I'm not sure how this picture was generated but it is not appropriate. Kicad, auto cad, etc are appropriate.